sopel/db.py

```diff
@@ -131,6 +131,12 @@ def alias_nick(self, nick, alias):
         except sqlite3.IntegrityError:
             raise ValueError('Alias already exists.')
 
+    def reset_nick_value(self, nick, key):
```

Comment:
As pointed out by @dgw the `delete` keyword is more appropriate when it comes to database operation.

And it's not the value that is reset, it's the key that is deleted.

```suggestion
    def delete_nick_key(self, nick, key):
```